Makefile

```diff
@@ -105,11 +108,18 @@ test_pytest:
 	@echo "Running py.test tests..."
 	py.test tests
 
+test_py: test_pylint test_flake8 test_doctest test_pytest test_other
+	@echo "Finished python and documentation tests!"
+
+test_shellcheck:
+	@echo "Running shellcheck..."
+	sed '2,$$s/^\( *\)#/\1/' ./ranger/data/scope.sh | shellcheck -a -
```

Comment:
This regex won't work for tabs. I know we shouldn't mix them in but it won't detect it either unless `shellcheck` handles that. Let's use `\s` instead.